### DISCUSSION TITLE
Please reinstate .tar.bz2 release artifact for v2.2.0

### DISCUSSION THREAD

**Discussion by paddatrapper** [savonet/liquidsoap#3285] [Installation] [answered]
The previous releases included a source code archive with all required generated files for building. v2.2.0 seems to be missing that. Last I checked, the `git archive` tarball was missing some generated files (#2521 for example).

Could this archive be reinstated? The compression type is not important, but having a source-code artifact that can be used to build liquidsoap is important for downstream maintainers

**Comment by toots** [comment, marked as answer]
Hi @paddatrapper 

The build system has changed with `v2.2.0`, we are now using `dune` and do not require generating a tarball with configure script and etc.

The release includes a tarball of the current code here: https://github.com/savonet/liquidsoap/archive/refs/tags/v2.2.0.tar.gz is that enough for you?
